bundles/org.openhab.binding.ojelectronics/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,158 @@
+				<default>1</default>
+			</parameter>
+			<parameter name="softwareVersion" type="integer" required="true">
+				<label>Software Version</label>
+				<description>Software Version</description>
+				<advanced>true</advanced>
+				<default>1060</default>
+			</parameter>
+		</config-description>
+	</bridge-type>
+	<thing-type id="owd5">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="ojcloud"/>
+		</supported-bridge-type-refs>
+		<label>OJ Electronics OWD5/MWD5 Thermostat</label>
+		<description>OJ Electronics OWD5/MWD5 Thermostat</description>
+		<category>RadiatorControl</category>
+		<channels>
+			<channel id="floorTemperature" typeId="floorTemperature"/>
+			<channel id="roomTemperature" typeId="roomTemperature"/>
+			<channel id="groupName" typeId="groupName"/>
+			<channel id="groupId" typeId="groupId"/>
+			<channel id="online" typeId="online"/>
+			<channel id="heating" typeId="heating"/>
+			<channel id="thermostatName" typeId="thermostatName"/>
+			<channel id="regulationMode" typeId="regulationMode"/>
+			<channel id="comfortSetpoint" typeId="comfortSetpoint"/>
+			<channel id="comfortEndTime" typeId="comfortEndTime"/>
+			<channel id="boostEndTime" typeId="boostEndTime"/>
+			<channel id="manualSetpoint" typeId="manualSetpoint"/>
+			<channel id="vacationEnabled" typeId="vacationEnabled"/>
+		</channels>
+		<config-description>
+			<parameter name="serialNumber" type="text" required="true">
+				<label>Serial Number</label>
+				<description>Serial number of the thermostat. You can find the serial number in the app or on the thermostat itself.</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+	<channel-type id="floorTemperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Floor Temperature</label>
+		<category>Temperature</category>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+	<channel-type id="groupName">
+		<item-type>String</item-type>
+		<label>Group Name</label>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="groupId">
+		<item-type>Number</item-type>
+		<label>Group ID</label>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="online">
+		<item-type>Contact</item-type>
+		<label>Online</label>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="heating">
+		<item-type>Contact</item-type>
+		<label>Heating</label>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="roomTemperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Room Temperature</label>
+		<category>Temperature</category>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+	<channel-type id="thermostatName">
+		<item-type>String</item-type>
+		<label>Thermostat Name</label>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="regulationMode">
+		<item-type>String</item-type>
+		<label>Regulation Mode</label>
+		<state readOnly="true">
+			<options>
+				<option value="1">Auto</option>
```

Comment:
Would it make sense to let the binding already map the numeric values to more meaningful strings, it will ease displaying on sitemaps and make rules etc more readable.

Reply:
I don't know what you mean exactly. I think this is the mapping of numeric value. Or did you mean i should use string in here and map in the handler in source code?

Reply:
> Or did you mean i should use string in here and map in the handler in source code?

Indeed sorry I was unclear :-)

Reply:
Done